packages/mjml-core/src/helpers/skeleton.js

```diff
@@ -11,8 +12,8 @@ export default function skeleton(options) {
     content = '',
     fonts = {},
     mediaQueries = {},
-    headStyle = [],
-    componentsHeadStyle = {},
```

Comment:
I updated these default values to match the types I was seeing during my testing, as well as what's defined here:

https://github.com/mjmlio/mjml/blob/988819de3375867c09585d28f555166b97415200/packages/mjml-core/src/index.js#L160-L161